uber/tests/conftest.py

```diff
@@ -52,6 +52,9 @@ def sensible_defaults():
     c.DEALER_BADGE_PRICE = 20
     c.PRICE_BUMPS = {}
 
+    # tests should turn this on to test the effects
+    c.HARDCORE_OPTIMIZATIONS_ENABLED = False
```

Comment:
I don't see what this does.  Does this have any effect on anything?  If we're resetting the cache every time, do we really need this?